src/roles.js

```diff
@@ -0,0 +1,21 @@
+const AccessControl = require('accesscontrol');
+
+const ac = new AccessControl();
+
+exports.roles = (function () {
+  ac.grant('donor')
```

Comment:
Can we have another role for the receiver too?
Or user role for both donor and receiver because a donor can be a receiver too